zc_utils.py

```diff
@@ -0,0 +1,19 @@
+import struct
+
+def write_compact_size(n):
+    if n < 253:
+        return struct.pack('B', n)
+    elif n < 0xFFFF:
+        return struct.pack('B', 253) + struct.pack('<H', n)
+    elif n < 0xFFFFFFFF:
+        return struct.pack('B', 254) + struct.pack('<I', n)
+    else:
+        return struct.pack('B', 255) + struct.pack('<Q', n)
+
+assert write_compact_size(0) == b'\x00'
+assert write_compact_size(252) == b'\xFC'
+assert write_compact_size(253) == b'\xFD\xFD\x00'
+assert write_compact_size(0xFFFE) == b'\xFD\xFE\xFF'
+assert write_compact_size(0xFFFF) == b'\xFE\xFF\xFF\x00\x00'
+assert write_compact_size(0xFFFFFFFE) == b'\xFE\xFE\xFF\xFF\xFF'
+assert write_compact_size(0xFFFFFFFF) == b'\xFF\xFF\xFF\xFF\xFF\x00\x00\x00\x00'
```

Comment:
`b'\xFE\xFF\xFF\xFF\xFF'

Also add tests for 0x10000, 0x100000000 (8 '0's), and 0xFFFFFFFFFFFFFFFF (16 'F's).